src/components/Post/Tabs/PostDescription.tsx

```diff
@@ -25,20 +29,76 @@ const CommentsContainer = dynamic(() => import('../Comment/CommentsContainer'),
 });
 
 interface IPostDescriptionProps {
+	allowed_roles?: string[] | null;
```

Comment:
allowed_roles should be used from context and not prop drilled

Reply:
resolved, pls check